kinetic/release-build.yaml

```diff
@@ -0,0 +1,62 @@
+%YAML 1.1
+# ROS buildfarm release-build file
+---
+abi_incompatibility_assumed: true
+jenkins_binary_job_priority: 78
```

Comment:
This must be a number numeric value then all other distros source jobs (Jade 80, Jade 81). Otherwise kinetic binarydeb jobs are blocking the sourcedeb jobs of the other distros.